requests/jupytercad-add-outputs.yml

```diff
@@ -0,0 +1,11 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - jupytercad: "jupytercad"
+  - jupytercad: "jupytercad-core"
+  - jupytercad: "jupytercad_core"
```

Comment:
are these underscores really necessary? with this kind of "output validation" you can "start off" with the "correct one".

They somewhat pollute environments and cause non-ssds to seriously slow down.

Reply:
oh ok. yeah you don't need to "add" these here since they are already "outputs".

You only need to add 'new outputs'